pkg/apiserver/profiling/model.go

```diff
@@ -47,6 +61,7 @@ type TaskGroupModel struct {
 	ProfileDurationSecs uint                          `json:"profile_duration_secs"`
 	TargetStats         model.RequestTargetStatistics `json:"target_stats" gorm:"embedded;embedded_prefix:target_stats_"`
 	StartedAt           int64                         `json:"started_at"`
+	ProfilingTypeList   string                        `json:"profiling_type_list"`
```

Comment:
Better to add a comment for it, documenting how the list is stored in a string. For example:

```go
// ProfilingTypeList is a comma separated list of profiling types requested by the user.
// Each profiling type is ensured to be a variant of `TaskProfilingType`.
```